wolfssl/wolfcrypt/settings.h

```diff
@@ -58,9 +58,16 @@
 /* Uncomment next line if using FreeRTOS */
 /* #define FREERTOS */
 
-/* Uncomment next line if using FreeRTOS+ TCP */
+/* Uncomment next line if using FreeRTOS+TCP */
 /* #define FREERTOS_TCP */
 
+	#ifndef min
```

Comment:
Where were you getting an undefined reference to min()?  We already define min() in several places if WOLFSSL_HAVE_MIN is not defined (ex: misc.c).